src/ui/user/UserIcons.js

```diff
@@ -8,44 +8,80 @@ type Props = {
   user: User,
 };
 
+type Icon = {
+  title: string,
+  value: string,
+};
+
+const getIcons = (user: User): Icon[] => {
+  const flags = user.flags || EMPTY_FLAGS;
+  const icons: Icon[] = [];
+
+  if (user.authLevel === "jr_admin") {
+    icons.push({
+      title: "Admin",
+      value: "⭐",
+    });
+  } else if (
+    user.authLevel === "sr_admin" ||
+    user.authLevel === "super_admin"
+  ) {
+    icons.push({
+      title: "Admin",
+      value: "🌟",
+    });
+  } else if (user.authLevel === "teacher") {
+    // icons.push({
```

Comment:
any reason not to uncomment these?

Reply:
I have no context why it was commented in the first place 🤷‍♂️ Personally I'd prefer to remove commented code completely. Can you suggest what to do with that case?

Reply:
uncomment and see if how well it works.  if it doesn't, then delete it.

Reply:
Looks like for Teachers, we already have a special label `Teacher`. It's not shown in the list view, but is visible in the detailed info. Do we need to duplicate it (teacher icon, teacher label)?

![image](https://user-images.githubusercontent.com/4667275/91664726-5b6b5480-eaf1-11ea-9bc3-f4137237721a.png)

Reply:
Same for playing, without icon in the list view it's not visible that user is playing, but is visible in detailed view:

![image](https://user-images.githubusercontent.com/4667275/91664782-aa18ee80-eaf1-11ea-8466-a456dc71697e.png)

Reply:
@leeschumacher ping

Reply:
I think I've lost the plot here.  Whats the downside to this?

Reply:
@steppefox It looks like we should uncomment the code because the "Teacher" and "Playing" icons wouldn't appear in the Chat list otherwise. The slight redundancy with icon and text does not hurt.

@leeschumacher Can we merge to `master`?

Reply:
Ah, ok, I'll rebase and uncomment tonight